static/js/echo.js

```diff
@@ -373,6 +373,14 @@ function handleRealmFilter(pattern, matches) {
     return url;
 }
 
+function handleTex(tex, fullmatch) {
+    try {
+        return katex.renderToString(tex);
+    } catch (ex) {
+        return '<span class="tex-error">' + fullmatch + '</span>';
```

Comment:
Wow, awesome catch! Hadn't thought about XSS, thank you very much for pointing that out.

I just pushed an updated commit, that should fix that problem.